### PR TITLE
Add documentation for contributing to this repo

### PR DESCRIPTION
**Pull request by loganmeetsworld**
# Add documentation for contributing to this repo

## 🗣 Description ##

This adds documentation from previous conversations we've had about PR etiquette and other rules for contributing here, especially helpful for onboarding.

### CODE REVIEW

**Review by SSPJ — APPROVED** [review]
LGTM 👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Conributing
```
Comment (h-m-f-t) [inline CONTRIBUTING.md:1]: Conributing -> Contributing

**Review by h-m-f-t — CHANGES_REQUESTED** [review]
👍 , flagged a small spelling error

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## What not to commit to the repo
+
+There are a handful of things we do not commit to the repository:
+- Information about particular vendors or procurement information
+- Anything related to PII whether it is from user interviews or feedback
+- User research related scheduling or documentation
```
Comment (colinmurphy01) [inline CONTRIBUTING.md:8]: I would like to add user research synthesis to our repo at some point to demonstrate our understanding. This doesn't need to be a blocker, but something to consider later on.

File: CONTRIBUTING.md (hunk 1 above)
Comment (loganmeetsworld) [reply CONTRIBUTING.md]: Thanks! Fixed

**Review by h-m-f-t — APPROVED** [review]
🇺🇸

**Pull request merged**